app/models/enrollment.rb

```diff
@@ -55,7 +55,7 @@ class Enrollment < ApplicationRecord
   accepts_nested_attributes_for :course_assignments, allow_destroy: true
 
   has_many :financial_aids, dependent: :destroy
-  has_many :travels, dependent: :destroy
+  # has_many :travels, dependent: :destroy
```

Comment:
The travel association should not be removed since we have travel records from past camps. If an associated record is deleted the travel records will be orphaned. I would not comment this line

Reply:
I agree, I returned the association to the model.